qemu/tests/virtio_fs_localfs_migration.py

```diff
@@ -0,0 +1,169 @@
+            else:
+                if params.get("viofs_svc_name", "VirtioFsSvc") == "VirtioFsSvc":
+                    error_context.context(
+                        "Start virtiofs service in guest.", test.log.info
+                    )
+                    debug_log_operation = params.get("debug_log_operation")
+                    if debug_log_operation:
+                        session = virtio_fs_utils.operate_debug_log(
+                            test, params, session, vm, debug_log_operation
+                        )
+                    virtio_fs_utils.start_viofs_service(test, params, session)
+                else:
+                    error_context.context(
+                        "Start winfsp.launcher instance in guest.", test.log.info
+                    )
+                    fs_volume_label = fs_params["volume_label"]
+                    start_multifs_instance(fs, fs_target, fs_volume_label)
+
+                # get fs dest for vm
+                virtio_fs_disk_label = fs_target
+                error_context.context(
+                    "Get Volume letter of virtio fs target, the disk"
+                    "lable is %s." % virtio_fs_disk_label,
+                    test.log.info,
+                )
+                vol_con = "VolumeName='%s'" % virtio_fs_disk_label
+                volume_letter = utils_misc.wait_for(
+                    lambda: utils_misc.get_win_disk_vol(session, condition=vol_con),
+                    params.get_numeric("cmd_timeout", 120),
+                )
+                if volume_letter is None:
+                    test.fail("Could not get virtio-fs mounted volume letter.")
+                fs_dest = "%s:" % volume_letter
+
+            guest_mnts[fs_target] = fs_dest
+        return session
+
+    def stop_service(session):
+        if os_type == "linux":
+            for fs_target, fs_dest in guest_mnts.items():
+                utils_disk.umount(fs_target, fs_dest, "virtiofs", session=session)
+                utils_misc.safe_rmdir(fs_dest, session=session)
+        else:
+            if params["viofs_svc_name"] == "WinFSP.Launcher":
+                for fs_target in guest_mnts.keys():
+                    error_context.context(
+                        "Unmount fs with WinFsp.Launcher.z", test.log.info
+                    )
+                    instance_stop_cmd = params["instance_stop_cmd"]
+                    session.cmd(instance_stop_cmd % fs_target)
+            else:
+                if guest_mnts:
+                    virtio_fs_utils.stop_viofs_service(test, params, session)
+
+    def test_migration(session):
+        # FIXME: Replace the vm's params to use a different shared virtio fs
+        vm.params["filesystems"] = vm.params["filesystems_migration"]
+        vm.migrate()
+        session = vm.wait_for_login()
+
+        for fs_dest in guest_mnts.values():
+            out = session.cmd_output(params["read_file_cmd"] % fs_dest).strip()
```

Comment:
The first step here should be to check the virtiofs mount status is still working well in the destination VM. Like to check that there is virtiofs type(like it's in the source VM) in the output of `mount` command in destination VM .